threads.php

```diff
@@ -375,3 +380,58 @@ function cfth_timeline_css() {
 <?php
 }
 
+// Meta field markup for add thread page
+function threads_taxonomy_add_new_meta_field() {
+	?>
+	<div class="form-field">
+		<label for="term_meta[order_term_meta]"><?php _e( 'Post order', 'threads' ); ?></label>
+			<select name="term_meta[order_term_meta]" id="term_meta[order_term_meta]">
+				<option value="ASC"><?php _e( 'ASC', 'threads' ); ?></option>
+				<option value="DESC"><?php _e( 'DESC', 'threads' ); ?></option>
+			</select>
+		<p class="description"><?php _e( 'Enter a value for this field','threads' ); ?></p>
+	</div>
+<?php
+}
+
+add_action( 'threads_add_form_fields', 'threads_taxonomy_add_new_meta_field', 10, 2 );
```

Comment:
In this action you've specified that our callback will accept 2 arguments but I see zero being captured in the function definition.